docs/release-process.md

```diff
@@ -23,8 +23,9 @@ Release shepherd responsibilities:
 
 | Release | Time of first RC     | Shepherd (GitHub handle)    |
 |---------|----------------------|-----------------------------|
-| v0.24.0 | (planned) 2021.09.28 | No one ATM                  |
-| v0.23.0 | 2021.09.01           | `@bwplotka`                 |
+| v0.25.0 | (planned) 2021.11.26 | No one ATM                  |
+| v0.24.0 | (planned) 2021.10.14 | No one ATM                  |
```

Comment:
BTW looking for volunteers (probably we should move this forward too)

Reply:
🐧i volunteer

Reply:
<3